### PR TITLE
WWST-4437 Add vid for this DTH and add Dooya fingerprint

### PR DESCRIPTION
**Pull request by BYDengS**
@greens and @dkirker , please help review this commit and merge it to china stg server. For the mnmn and vid, my colleague has created right uimetadata to STG server. Thank you :)

### CODE REVIEW

File: devicetypes/smartthings/zigbee-curtain.src/zigbee-curtain.groovy [hunk 1]
```diff
@@ -115,13 +131,25 @@ def ping() {
 
 def refresh() {
     log.info "refresh()"
-    return zigbee.readAttribute(zigbee.LEVEL_CONTROL_CLUSTER, ATTRIBUTE_CURRENT_LEVEL)
+    def cmds
+    if (device.getDataValue("manufacturer") == "REXENSE") {
```
Comment (greens) [inline devicetypes/smartthings/zigbee-curtain.src/zigbee-curtain.groovy]: this manufacturer check should be a separate function

File: devicetypes/smartthings/zigbee-curtain.src/zigbee-curtain.groovy [hunk 2]
```diff
@@ -23,12 +23,14 @@ metadata {
         capability "Stateless Curtain Power Button"
         capability "Window Shade"
 
-        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0102", outClusters: "000A", manufacturer: "Feibit Co.Ltd", model: "FTB56-ZT218AK1.6", deviceJoinName: "Wistar Curtain Motor(CMJ)" // SY-IoT201-BD
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0102", outClusters: "000A", manufacturer: "Feibit Co.Ltd", model: "FTB56-ZT218AK1.6", deviceJoinName: "Wistar Curtain Motor(CMJ)"
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0102", outClusters: "0003",manufacturer: "REXENSE", model: "DY0010", deviceJoinName: "Smart Curtain Motor(DT82TV)"
```
Comment (greens) [inline devicetypes/smartthings/zigbee-curtain.src/zigbee-curtain.groovy]: nit: missing a space before `manufacturer`

**Comment by dkirker** [comment]
@BYDengS I left a comment on the now merged PR #6841 I know it was already merged, but if the Wistar device hasn't be certified yet, it might be a good idea to combine the device handlers.

**Comment by BYDengS** [comment]
@greens and @dkirker , please help review this commit and merge it to china stg server. I removed the dooya DTH from `zigbee-curtain` DTH, and add it to `zigbee-window-shade` DTH.

**Review by dkirker — APPROVED** [review]

**Pull request merged**